db/seeds/organisations.rb

```diff
@@ -0,0 +1,13 @@
+require "yaml"
+
+organisations = YAML.safe_load(File.read(File.join(Rails.root, "db", "seeds", "organisations.yml")))
+organisations.each do |organisation|
+  organisation_params = {
+    name: organisation["name"],
+    organisation_type: organisation["type"],
+    language_code: "en",
+    default_currency: "gbp",
+    service_owner: organisation["service_owner"],
+  }
+  Organisation.find_or_create_by(iati_reference: organisation["reference"]).update(organisation_params)
```

Comment:
I like the way this is implemented as we should be able to add more records through our seeds idempotently.